robotbase/src/main/java/com/team2357/lib/commands/setFailsafeCommand.java

```diff
@@ -0,0 +1,7 @@
+package com.team2357.lib.commands;
+
+// TODO: ShuffleBoard Implement
```

Comment:
This seems irrelevant with the changes made to the failSafeCommand, and would have the same functionality. This can be deleted.

Reply:
Still need to delete this.